src/index.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+Hello, world!
```

Comment:
always add an empty line at the end of the file, there is a linter rule for this, try to config it